packages/jellyfish-transaction/__tests__/script/dftx/dftx_vault/WithdrawFromVault.test.ts

```diff
@@ -0,0 +1,82 @@
+import { SmartBuffer } from 'smart-buffer'
+import {
+  CWithdrawFromVault,
+  WithdrawFromVault
+} from '../../../../src/script/dftx/dftx_vault'
+import { OP_CODES } from '../../../../src/script'
+import { toBuffer, toOPCodes } from '../../../../src/script/_buffer'
+import { OP_DEFI_TX } from '../../../../src/script/dftx'
+import BigNumber from 'bignumber.js'
+
+it('should bi-directional buffer-object-buffer', () => {
+  const fixtures = [
+    /**
+     * WithdrawFromVault : {
```

Comment:
Just for knowledge sharing, we need to add at least 3 fixtures for `should bi-directional buffer-object-buffer`
At you are copied this from the previous programmer, we can ignore this for now.